lib/netsuite/version.rb

```diff
@@ -1,3 +1,3 @@
 module Netsuite
-  VERSION = '0.7.3'
+  VERSION = '0.7.5'
```

Comment:
do we either want to introduce a new version convention, or maybe rename the gem in gemspec to avoid future confusion (specifically, version collision against the public gem) if we decide to use the fork?

Reply:
We can specify the gem via the git url in the Gemfile, which should hopefully be enough to prevent confusion. I don't have a better way around versioning rather than just "use whatever's on master on our fork"... I _think_ we can update that to use tags, so I'll take a look at doing that once we switch the Gemfile over to use our fork.

Reply:
could we still turn this into a garbage string or something if we are basically using it as a library? that way it doesn't show up confusingly in logs, etc.

Reply:
conventionally it might be something like `0.7.5.flexport.1` or something like that